sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -209,6 +210,18 @@ public EventHubClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Sets the Event Hub connection to use when interacting with Event Hubs. If not set, a new connection will be
+     * constructed and used.
+     *
+     * @param eventHubConnection Event Hub connection to use.
+     * @return The updated {@link EventHubClientBuilder} object.
+     */
+    public EventHubClientBuilder connection(EventHubConnection eventHubConnection) {
```

Comment:
Check for null.

Reply:
In general, I think we do validation on build, so if they set this to null, we will just create a new connection for them.